weblate/trans/forms.py

```diff
@@ -847,19 +847,19 @@ def clean(self):
                 pk=self.cleaned_data["revert"], unit=self.unit
             )
         except Change.DoesNotExist:
-            raise ValidationError(_("Could not find reverted change."))
+            raise ValidationError(_("Could not find the reverted change."))
         return self.cleaned_data
 
 
 class AutoForm(forms.Form):
-    """Automatic translation form."""
+    """Auto-translation form."""
 
     mode = forms.ChoiceField(
-        label=_("Automatic translation mode"),
+        label=_("Auto-translation mode"),
```

Comment:
To me, "automatic translation" sounds better. It is also used in other strings (for example, few lines below this one).

Reply:
```suggestion
        label=_("Automatic translation mode"),
```